wolfssl/wolfcrypt/types.h

```diff
@@ -547,6 +549,7 @@ decouple library dependencies with standard string, memory and so on.
             VAR_TYPE VAR_NAME[VAR_SIZE]
         #define WC_DECLARE_ARRAY(VAR_NAME, VAR_TYPE, VAR_ITEMS, VAR_SIZE, HEAP) \
             VAR_TYPE VAR_NAME[VAR_ITEMS][VAR_SIZE]
+        #define WC_INIT_ARRAY(VAR_NAME, VAR_TYPE, VAR_ITEMS, VAR_SIZE, HEAP) do {} while(0)
```

Comment:
Is the `do {} while(0)` really needed?

Reply:
I too have my doubts about the `do {} while(0)`, but AFAIK it's the only universally safe way to swallow a semicolon. we use it elsewhere -- although upon review, only John and I do it elsewhere :-)